app/src/main/java/org/oppia/android/app/options/LanguageItemViewModel.kt

```diff
@@ -1,9 +1,19 @@
 package org.oppia.android.app.options
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Transformations
 import org.oppia.android.app.viewmodel.ObservableViewModel
 
 /**
  * Language item view model for the recycler view in [AppLanguageFragment] and
  * [AudioLanguageFragment].
  */
-class LanguageItemViewModel(val language: String) : ObservableViewModel()
+class LanguageItemViewModel(
+  val language: String,
+  val selectedLanguage: LiveData<String>,
```

Comment:
Can this be private if it's being modified in the parent view model?

Reply:
Done